src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -23,9 +23,17 @@ col-sm-2
 {# Rows #}
 
 {% block form_row -%}
-    {%- set widget_attr = {} -%}
+    {%- set describedby = '' -%}
+    {%- if errors|length > 0 -%}
+        {%- set describedby = (describedby~" form-error-"~id) | trim -%}
```

Comment:
`describedby` will always be empty. No need for concatenation.

Reply:
good catch

Reply:
Concatenation is still here, right?